recipes/stochatreat/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "stochatreat" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1ef72f04a94259c7b3beb6de9eb4facfb04b30e2e7ca2b8ec8c3c20588143b04
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - typing
+    - pandas
+    - numpy
+  run:
+    - python
+    - pip
+    - setuptools
+    - typing
+    - pandas
+    - numpy
+
+test:
+  imports:
+    - stochatreat
+  source_files:
```

Comment:
You need to add those to the recipe to run them but import tests are fine in conda packages, so just skip this.